package.json

```diff
@@ -44,6 +46,9 @@
     "@sap-cloud-sdk/http-client": "^3.21.0",
     "@sap-cloud-sdk/openapi-generator": "^3.21.0",
     "@sap-cloud-sdk/util": "^3.21.0",
+    "@cap-js/sqlite": "^1",
```

Comment:
Is the DB really needed, considering that we currently don't store any embeddings?